contracts/modules/Checkpoint/Voting/APLCR/AdvancedPLCRVotingCheckpoint.sol

```diff
@@ -294,8 +294,12 @@ contract AdvancedPLCRVotingCheckpoint is AdvancedPLCRVotingCheckpointStorage, Vo
     function _addExemptedAddresses(address[] memory _exemptedAddresses, uint256 _ballotId) internal {
         for (uint256 i = 0; i < _exemptedAddresses.length; i++) {
             Ballot storage ballot = ballots[_ballotId];
-            if (_exemptedAddresses[i] != address(0)) 
+            if (_exemptedAddresses[i] != address(0))
                 ballot.exemptedVoters[_exemptedAddresses[i]] = true;
+            // @FIXME uncomment this once we decrease contract size.
+            // else {
+            //     delete _exemptedAddresses[i];
```

Comment:
delete makes that particular index value will be 0 and in our case, it is automatically 0 so it doesn't make sense too much